src/vueShims.d.ts

```diff
@@ -1,5 +1,5 @@
 declare module '*.vue' {
   // TODO: Figure out the typing for this
-  import Vue from 'vue'
-  export default any
+  const component: any
+  export default component
```

Comment:
unrelated: Maybe we should use the same shim as the one provided by the CLI. It is not cast as `any` I know, but that would help catch errors that developers will encounter.

Reply:
I think `Component` is a better candidate, but needs testing

Reply:
From what I can tell since the shim gives you no information about your component (eg props/data/whatever) the majority of the types we have are not very useful. in my experience the vast majority of people are using `vue` files.

I think we are focusing too heavily on "correct typing" and not enough on the actual DX. At least from my point of view, I would rather not have type safety than have random ts compile errors even doing the most basic of things (like using `data`). 

I wonder if there is some way to ignore all the mounting types when the imported component is from a `vue` file? That would really be ideal, at least at this point, since you cannot get any useful information from an SFC right now.

Reply:
without a typescript plugin that will handle vue files, I doubt we can extract much information from a `.vue` file.

The shim only tells the typescript: "this file import is of this type" - without any process.

Reply:
I think we need to merge something (this or something better?) sooner rather that later... using `data` to set a component up for testing is basically the Hello World of VTU and it doesn't work out of the box atm. I do not want to ruin the hard work that has gone into typing VTU, though. 🤔